### DISCUSSION TITLE
Get all sign in the world

### DISCUSSION THREAD

**Discussion by Timofffee** [Minestom/Minestom#1552] [Help]
Hello. I upload Anvil map and want to process it after that. I use tablets to store some data. For example, where the player appears and where some entity should stand.

What is the easiest way I can get all the tablets in the world, read their contents and then delete them from the world?

**Comment by Obyvante** [comment]
The easiest way to implement your sign handler code/logic into the anvil loader class and chunk load method. You will see the method is loading blocks. Thus, you will be able to process if the block is signed and then continue without placing a sign.

https://github.com/Minestom/Minestom/blob/e5d0a43ba42c7023a93e8e33356909e614783ac7/src/main/java/net/minestom/server/instance/AnvilLoader.java#L218-L271
